src/mscorlib/src/System/Threading/CancellationToken.cs

```diff
@@ -40,24 +29,21 @@ namespace System.Threading
     [DebuggerDisplay("IsCancellationRequested = {IsCancellationRequested}")]
     public struct CancellationToken
     {
+        private readonly static Action<object> s_actionToActionObjShunt = obj => ((Action)obj)();
```

Comment:
Yes.  Though even then we might still want to do this to avoid the laziness / null check.